### ISSUE TITLE
Some links cannot be read because of the color contrast in dark mode

### ISSUE THREAD

**Issue by etiennebacher** [astral-sh/ruff#8519]
In the [Vim & Neovim section](https://docs.astral.sh/ruff/integrations/#vim-neovim), some links are unreadable because of their color in dark mode:

![image](https://github.com/astral-sh/ruff/assets/52219252/a4587c42-ac18-4cdb-bbfe-0da79d7fefcf)

Note that everything's fine in light mode:

![image](https://github.com/astral-sh/ruff/assets/52219252/cabfa660-ef63-454b-a899-665ac89656b7)

**Comment by charliermarsh** [comment]
Thank you!
